src/libraries/System.Private.CoreLib/src/System/MissingMemberException.cs

```diff
@@ -70,6 +70,6 @@ public override string Message
         // format depending on the language environment.
         protected string? ClassName;
         protected string? MemberName;
-        protected byte[]? Signature;
+        protected byte[]? Signature; // Unused in coreclr
```

Comment:
```suggestion
        protected byte[]? Signature; // Unused
```
It is unused in Mono too.